### ISSUE TITLE
Kodi pauses when HDMI display is powered off (Nvidia Shield TV pro)

### ISSUE THREAD

**Issue by spfma-tech** [xbmc/xbmc#25109]
### Describe the bug
Here is a clear and concise description of what the problem is:

Kodi seems to pause when the associated HDMI display is powered off (or by  unplugging the cable).
As soon as the HDMI device is back, audio and video are playing again exactly where it stopped.
None of the audio playing apps I have tried (YT, Deezer, some online radios) exhibits this behavior, switching off the display does not disturb them.
I am using an amplifier with integrated class compliant USB  DAC so it has nothing to do with HDMI connectivity (not an AVR with HDMI hub or ARC)
#Expected Behavior
Here is a clear and concise description of what was expected to happen:

Kodi should not stop playing when then associated HDMI device is off.

## Actual Behavior

Kodi pauses, just as it does when i open Nvidia settings menu while playing a video.

## Possible Fix

Maybe some advanced setting should give control on a way to ignore HDMI events.

### To Reproduce

1.  Put some source with audio to play
2. Switch the screen off or remove the cable
3.

### Debuglog

The debuglog can be found here: https://paste.kodi.tv/aseworuyev.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [X ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Shield Android TV 11
 - Kodi version: 21.0.1 (was also present on previous versions)

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by neo1973** [comment]
Go to Settings -> System -> Input -> Peripherals -> CEC. One of the settings should do the trick. For more assistance please head to to the forum, this is purely a bug tracker.

**Comment by spfma-tech** [comment]
So this is what I once saw in a forum but could not find again.

On the Shield there is nothing in this peripherals list. 

I have tried with and without CEC at device config level.

The example I saw was for another kind of box and there were several options.

So maybe my problem is I just can’t change default values because of that ? 

It just gave me the idea to bring a simple computer to my Shield, it has no CEC capabilities. Guess what, the problem does not occur.
